Mathlib/MeasureTheory/SetAlgebra.lean

```diff
@@ -0,0 +1,227 @@
+/-
+Copyright (c) 2024 Etienne Marion. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Etienne Marion
+-/
+import Mathlib.MeasureTheory.SetSemiring
```

Comment:
Is there a link between `IsSetSemiring` and `IsSetAlgebra` that is easy to formalize?